app/views/projects/_no_shared_runners.html.haml

```diff
@@ -0,0 +1,5 @@
+.alert.alert-danger
+  %p
+    There are NO runners to build this project. Consider assigning this project to
+    %span.label.label-info a specific
```

Comment:
You're right. It doesn't make sense. I just wanted to have the same label
as in runners admin view.


On Fri, Jan 2, 2015 at 3:59 PM, Drew Blessing <notifications@github.com>
wrote:

> In app/views/projects/_no_shared_runners.html.haml
> <https://github.com/gitlabhq/gitlab-ci/pull/488#discussion-diff-22411917>:
>
> > @@ -0,0 +1,5 @@
> > +.alert.alert-danger
> > +  %p
> > +    There are NO runners to build this project. Consider assigning this project to
> > +    %span.label.label-info a specific
>
> What does this look like in the UI? Does it make sense to have a specific
> inside the label or on the line before? Same with the specific runner
> partial.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gitlabhq/gitlab-ci/pull/488/files#r22411917>.
>



-- 
Kamil Trzciński

ayufan@ayufan.eu
www.ayufan.eu